components/chosenArticlePage/VotingContainer.jsx

```diff
@@ -0,0 +1,47 @@
+import { useEffect, useState } from 'react';
+
+import Button from 'react-bootstrap/Button';
+import patchArticleVotes from '../../utils/patchArticleVotesByArticleId';
+
+
+export default function VotingContainer ({votes, setVotes, chosenArticleId}) {
+
+    const [updateVotes, setUpdateVotes] = useState([])
```

Comment:
You initialise this state to an array but later set it to an object, it's best to be consistent and stick to the same datatype